packages/vuetify/src/components/VTimeline/VTimeline.sass

```diff
@@ -18,36 +21,47 @@
 
   @include horizontal(true)
     grid-template-rows: $timeline-item-grid-template
+    grid-column-gap: $timeline-item-padding
     width: 100%
 
-    .v-timeline-item__body, .v-timeline-item__opposite
-      padding-inline-start: calc((var(--v-timeline-dot-size) / 2) - 5px)
+    .v-timeline-item:first-child
+      .v-timeline-item__body, .v-timeline-item__opposite, .v-timeline-divider
+        padding-inline-start: $timeline-item-padding
+
+    .v-timeline-item:last-child
+      .v-timeline-item__body, .v-timeline-item__opposite, .v-timeline-divider
+        padding-inline-end: $timeline-item-padding
 
     .v-timeline-item:nth-child(2n)
       .v-timeline-item__body
         grid-row: 3
-        padding-top: $timeline-item-padding
+        padding-block-start: $timeline-item-padding
```

Comment:
Any reason for using block properties? We use inline for rtl support, but nothing works with vertical writing modes so it's simpler to stick to top/bottom.

Reply:
It was mostly to have some symmetry in the css